docs/contents/dev-docs/testing.md

```diff
@@ -246,3 +248,57 @@ make delete-all-kind-clusters
 
 With this the testing is completed. Thanks for your time and we appreciate your
 feedback.
+
+## Installing helm charts from ECR repository and test on Amazon EKS cluster
+This section contains steps to test ACK service controller using helm chart and Amazon EKS cluster.
+These steps take ACK ElastiCache controller as example and require Amazon EKS cluster. 
```

Comment:
nit: "These steps take the ACK ElastiCache controller as an example and require an Amazon EKS cluster"